src/commands/utils/tempfile.ts

```diff
@@ -16,9 +16,9 @@ export async function withOptionalTempFile<T>(
     }
 }
 
-export async function createTempFile(content: string, extension: string): Promise<TempFile> {
+export async function createTempFile(content: string, extension: string, prefix?: string): Promise<TempFile> {
```

Comment:
💡 Note to self: let's extract this to separate function which can be used for more broader reasons.

Reply:
Let me add this in this PR, will be much better for the entirety of this PR.